Description: Blacklist Canon EOS 3D for now, it leads to a SIGSEGV in libc.
Author: Alessio Treglia <alessio@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=687004
---
 util/mtp-hotplug.c |    2 ++
 1 file changed, 2 insertions(+)

--- libmtp.orig/util/mtp-hotplug.c
+++ libmtp/util/mtp-hotplug.c
@@ -148,6 +148,8 @@ int main (int argc, char **argv)
       printf("ATTR{idVendor}==\"0971\", GOTO=\"libmtp_rules_end\"\n");
       printf("# Canon scanners that look like MTP devices (PID 0x22nn)\n");
       printf("ATTR{idVendor}==\"04a9\", ATTR{idProduct}==\"22*\", GOTO=\"libmtp_rules_end\"\n");
+      printf("# Canon digital camera (EOS 3D) that looks like MTP device (PID 0x3113)\n");
+      printf("ATTR{idVendor}==\"04a9\", ATTR{idProduct}==\"3113\", GOTO=\"libmtp_rules_end\"\n");
       printf("# Sensitive Atheros devices that look like MTP devices\n");
       printf("ATTR{idVendor}==\"0cf3\", GOTO=\"libmtp_rules_end\"\n");
       printf("# Sensitive Atmel JTAG programmers\n");
